### ISSUE TITLE
NVDA not reading any browse results

### ISSUE THREAD

**Issue by poonamdeo** [nvaccess/nvda#12217]
### Steps to reproduce:
1. Open Browser (Chrome, Firefox, Microsoft Edge)
2. Browse any search result
3. Press Enter
### Actual behavior:
NVDA does not read words after being typed and also does not read the page loaded, instead announces as unknown and sometimes doesn't announce anything.
### Expected behavior:
NVDA should read the words typed and after completing the search it should read the open window.
### System configuration
Windows 10 Home Single Language
64-bit Operating System, x64-based Processor
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Version: 2020.4
#### Windows version:
Windows 10 version 10.0.18363 Build 18363
#### Name and version of other software in use when reproducing the issue:
Chrome Version 89.0.4389.90 (Official Build) (64-bit)
Firefox Version 86.0 (64-bit)
Microsoft Edge Version 89.0.774.57 (Official build) (64-bit)
#### Other information about your system:
Running Windows 10
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA 2018.1 - Reading as unknown
NVDA 2018.2 - Reading as unknown
NVDA 2019.1 - Reading as unknown
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu/tools?
Yes
[nvda.log](https://github.com/nvaccess/nvda/files/6191902/nvda.log)

**Comment by Brian1Gaff** [comment]
Have you recently uninstalled an office product or any other major 
application or anti virus system? What anti virus do you use.

I've seen this in windows 7 as well, and it tends to be some registry 
corruption or incorrect restoration by an uninstaller. I took my machine 
back to an earlier date to fix it and eventually found it was avast, which I 
used Avastclear to remove in safe mode, I understood they had now fixed this 
problem though.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message ----- 
From: "Poonam Deokar" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Tuesday, March 23, 2021 6:25 PM
Subject: [nvaccess/nvda] NVDA not reading any browse results (#12217)

**Comment by poonamdeo** [comment]
@Brian1Gaff 
I have recently uninstalled Microsoft Office Professional 2010 and installed Microsoft Office Home and Student 2019.
The antivirus I have on my system is Seqrite Endpoint Security, product version 17.00 (10.3.1.6), 64-bit.
Windows got updated to version 2021-03 Cumulative Update Preview for Windows 10 Version 20H2 for x64-based Systems (KB5000842)
What is the next step to check registry corruption?
​​​​​​​

**Comment by ehollig** [comment]
Hello @poonamdeo, is this issue still occurring with latest version of NVDA and web browsers? Also, could you provide more information on the steps that you took to "browse the search results"?

**Comment by poonamdeo** [comment]
Hello Ethan,

This issue is sorted with a Windows upgrade.
Thank you for checking in!

Regards,
Poonam Deokar
Co-ordinator for Students with Disabilities, St. Xavier's College
(Autonomous),
Sr. Consultant - Education, Xavier's Resource Centre for the Visually
Challenged (XRCVC <http://xrcvc.org/>)
St. Xavier's College (Autonomous),
5, Mahapalika Marg,
Mumbai-400 001
# 022-22623298
